tests/tests_pytorch/trainer/test_supporters.py

```diff
@@ -265,26 +245,6 @@ def test_combined_loader_sequence_max_size_cycle():
     assert idx == len(combined_loader) - 1
 
 
-@pytest.mark.parametrize(
-    ["input_data", "compute_func", "expected_length"],
-    [
-        ([*range(10), list(range(1, 20))], min, 0),
-        ([*range(10), list(range(1, 20))], max, 19),
-        ([*range(10), {str(i): i for i in range(1, 20)}], min, 0),
-        ([*range(10), {str(i): i for i in range(1, 20)}], max, 19),
-        ({**{str(i): i for i in range(10)}, "nested": {str(i): i for i in range(1, 20)}}, min, 0),
-        ({**{str(i): i for i in range(10)}, "nested": {str(i): i for i in range(1, 20)}}, max, 19),
-        ({**{str(i): i for i in range(10)}, "nested": list(range(20))}, min, 0),
-        ({**{str(i): i for i in range(10)}, "nested": list(range(20))}, max, 19),
-    ],
-)
-def test_nested_calc_num_data(input_data, compute_func, expected_length):
```

Comment:
I tried to leave tests mostly untouched. But this one I didn't know what would be the equivalent version